app/controllers/citizens/student_finances/annual_amounts_controller.rb

```diff
@@ -20,7 +20,7 @@ def update
       private
 
       def irregular_income
-        @irregular_income ||= IrregularIncome.new
+        legal_aid_application.irregular_incomes.find_by(income_type: 'student_loan')
```

Comment:
Just thinking about if there'll be a case where there is no irregular income.

As in, should it be 
```ruby
@irregular_income ||= (legal_aid_application.irregular_incomes.find_by(income_type: 'student_loan') || IrregularIncome.new)
```